fastcrypto/src/groups/ristretto255.rs

```diff
@@ -136,16 +149,29 @@ impl Mul<RistrettoScalar> for RistrettoScalar {
     }
 }
 
+impl Div<RistrettoScalar> for RistrettoScalar {
+    type Output = RistrettoScalar;
+
+    fn div(self, rhs: RistrettoScalar) -> RistrettoScalar {
+        // TODO: Is there a better way?
+        assert_ne!(rhs.0, ExternalRistrettoScalar::zero());
```

Comment:
Did you check that? https://github.com/dalek-cryptography/curve25519-dalek/blob/90292dccf84cdc614c1a5640f2d6cb13bfff9761/src/field.rs#L106

Reply:
The todo is actually regarding replacing the assert with something better that doesn't panic.
I don't see how to do that as long as we use the Div trait.

Reply:
Oh I see, then do we need to even check or panic? This might be adversarial input and while we're in a cyclic group, dividing by zero is actually multiplying with the invert and we might decide to actually allow it or we always assume our RistrettoScalar serialization never serializes a zero object, so this will never happen.

Reply:
Right, we should use it like Optional's unwrap, i.e., only when we know we are safe.
I included the assert_ne call just to make it make explicit and less dependent on the external library.